website/docs/r/report_rule.html.markdown

```diff
@@ -0,0 +1,138 @@
+For more information, see the [Report Rules documentation](https://docs.lacework.com/report-rules).
+
+## Example Usage
+
+#### Report Rule with Aws Compliance Reports
+```hcl
+resource "lacework_report_channel_email" "team_email" {
+  name       = "Team Emails"
+  recipients = ["foo@example.com", "bar@example.com"]
+}
+
+resource "lacework_report_rule" "example" {
+  name                 = "My Report Rule"
+  description          = "This is an example report rule"
+  email_alert_channels = [lacework_report_channel_email.team_email.id]
+  severities           = ["Critical", "High"]
+
+  aws_compliance_reports {
+    cis_s3 = true
+  }
+  weekly_snapshot = true
+}
+```
+
+#### Report Rule with Gcp Compliance Reports and Gcp Resource Group
+```hcl
+resource "lacework_report_channel_email" "team_email" {
+  name       = "Team Emails"
+  recipients = ["foo@example.com", "bar@example.com"]
+}
+
+resource "lacework_resource_group_gcp" "all_gcp_projects" {
+  name         = "GCP Resource Group"
+  description  = "All Gcp Projects"
+  organization = "MyGcpOrg"
+  projects     = ["*"]
+}
+
+resource "lacework_report_rule" "example" {
+  name             = "My Report Rule"
+  description      = "This is an example report rule"
+  channels         = [lacework_report_channel_slack.ops_critical.id]
+  resource_groups  = [lacework_resource_group_gcp.all_gcp_projects.id]
+  severities       = ["Critical"]
+
+  gcp_compliance_reports {
+    k8s = true
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The report rule name.
+* `channels` - (Required) The list of report channels for the rule to use.
+* `severities` - (Required) The list of the severities that the rule will apply. Valid severities include: 
+  `Critical`, `High`, `Medium`, `Low` and `Info`.
+* `description` - (Optional) The description of the report rule.
+* `resource_groups` - (Optional) The list of resource groups the rule will apply to.
+* `enabled` - (Optional) The state of the external integration. Defaults to `true`.
+* `aws_compliance_reports` - (Optional) Compliance reports for Aws. See [Aws Compliance Reports](#Aws Compliance Reports) below for details.
+* `azure_compliance_reports` - (Optional) Compliance reports for Azure. See [Azure Compliance Reports](#Azure Compliance Reports) below for details.
+* `gcp_compliance_reports` - (Optional) Compliance reports for Gcp. See [Gcp Compliance Reports](#Gcp Compliance Reports) below for details.
+* `daily_compliance_reports` - (Optional) Daily event summary reports. See [Daily Compliance Reports](#Daily Compliance Reports) below for details.
+* `weekly_snapshot` - (Optional) A weekly compliance trend report for all monitored resources. Defaults to `false`.
+
+### Aws Compliance Reports
+
+`aws_compliance_reports` supports the following arguments:
+
+* `cis_s3` - (Optional) Defaults to `false`.
```

Comment:
could you please add a long description in both places, here in the website documentation as well as every field inside the Schema? You can get the actual report names/sentences from the UI:
![Screen Shot 2021-11-29 at 7 45 57 PM](https://user-images.githubusercontent.com/5712253/143982366-cbe5d085-5777-4e78-9d51-8a03b62e7407.png)

Reply:
👍